src/util/LinkUtils.ts

```diff
@@ -1,3 +1,13 @@
+const isRefererInternal = (referer: string, host: string) => {
+  return referer && referer.includes(host);
+};
+
 export const getBackUrl = (referer: string, host: string) => {
-  return referer && referer.includes(host) ? referer : "/";
+  return isRefererInternal(referer, host) ? referer : "/";
+};
+
+export const getBackToSearchUrl = (referer: string, host: string) => {
```

Comment:
Maybe worth checking in with Sean:  Is it better to make this link disappear if you didn't navigate from search, or is it better to call it something like "back to search" and take you to home if you didn't come from search?

Reply:
OK.  The only weird thing here is that if you navigate from search to an item, and then refresh the page, the page looks different after refresh.  I wasn't able to test this on research catalog because I couldn't make the "back to search results" link appear, but wanted to confirm that this is OK?

Reply:
That matches the behavior on research catalog.